tests/unit/ssg-module/data/draft_oval.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
```

Comment:
What about editing the OVAL, so it contains less verbose and more versatile data? For example, the ID could be rewritten to `RULE_ID`, and things that are not tested and not required by the build system code for smooth finish could get removed from the XML file.